DS13/code/game/movietitles.dm

```diff
@@ -0,0 +1,83 @@
+//Makes end credits happen after round end!
+//Inspired by baystation, optimized for /tg/
+
+/datum/asset/simple/credits
+	assets = list("credits.gif"	= 'DS13/icons/stars2.gif')
+
+/proc/names2credits()
+	var/credits = "Starring the crew of [station_name()]<br>"
+	for(var/mob/living/M in GLOB.player_list)
+		if(!isliving(M) || !M.mind || !M.client)
+			continue //Ghosts don't get credit
+		credits += "[M.client.ckey] as [M.mind.assigned_role] [M.name]<br>"
+	credits += "Directed by: Kmc2000 <br> Executive producer: Francinum <br> Technical supervisor: Bass-ic <br>"
+	return credits
+
+/proc/roll_credits()
+	var/text = names2credits()
+	SEND_SOUND(world, 'DS13/sound/effects/endcredits.ogg')
+	for(var/mob/M in GLOB.player_list)
```

Comment:
loop through clients connected via `for(var/client/C in GLOB.clients)` instead of looping through mobs so ghosts also see the credits